samples/wifi/wfa_qt_app/src/qt_app_main.c

```diff
@@ -16,15 +16,64 @@
 #include <zephyr/device.h>
 #include <zephyr/net/net_config.h>
 
+#ifdef CONFIG_USB_DEVICE_STACK
+#include <zephyr/usb/usb_device.h>
+
+#define CONFIG_NET_CONFIG_USB_IPV4_ADDR "10.20.30.40"
+int init_usb(void)
+{
+	int ret;
+
+	ret = usb_enable(NULL);
+	if (ret != 0) {
+		printk("Cannot enable USB (%d)", ret);
+		return ret;
+	}
+
+	return 0;
+}
+#endif /* CONFIG_USB_DEVICE_STACK */
+
 int main(void)
 {
+#ifdef CONFIG_USB_DEVICE_STACK
+	struct in_addr addr;
+#endif /* CONFIG_USB_DEVICE_STACK */
+
 #if defined(CLOCK_FEATURE_HFCLK_DIVIDE_PRESENT) || NRF_CLOCK_HAS_HFCLK192M
 	/* For now hardcode to 128MHz */
 	nrfx_clock_divider_set(NRF_CLOCK_DOMAIN_HFCLK,
 			       NRF_CLOCK_HFCLK_DIV_1);
 #endif
 	printk("Starting %s with CPU frequency: %d MHz\n", CONFIG_BOARD, SystemCoreClock/MHZ(1));
 
+#ifdef CONFIG_USB_DEVICE_STACK
+	init_usb();
+	/* Redirect static IP address to netusb*/
+	const struct device *usb_dev = device_get_binding("eth_netusb");
+	struct net_if *iface = net_if_lookup_by_dev(usb_dev);
+
+	if (!iface) {
+		printk("Cannot find network interface: %s", "eth_netusb");
+		return -1;
+	}
+	if (sizeof(CONFIG_NET_CONFIG_USB_IPV4_ADDR) > 1) {
```

Comment:
You can use the default static IP address for USB and DHCP for Wi-Fi.

Reply:
In QT, DHCP configuration is not used. So, keeping static configuration only.